Dockerfile

```diff
@@ -14,8 +14,9 @@ ENV GOPATH /go
 COPY --from=0 /go/ $GOPATH
 ENV PATH $PATH:$GOPATH/bin
 
-RUN apk add --update python python-dev py-pip build-base && \
-    pip install enum34 retrying google-api-core google-cloud-storage && \
+RUN apk add --update python3 python-dev py-pip build-base && \
```

Comment:
Umm, I think it's supposed to be python3-dev - https://pkgs.alpinelinux.org/package/v3.3/main/x86/python3-dev